modules/ROOT/pages/dw-core-functions-write.adoc

```diff
@@ -6,16 +6,18 @@ endif::[]
 
 
 [[write1]]
-== write&#40;Any, String, Object&#41;: String &#124; Binary
+== write&#40;value: Any, contentType: String = "application/dw", writerProperties: Object = {}&#41;: String &#124; Binary
 
 Writes a value as a string or binary in a supported format.
 
 
-Returns a string or binary with the serialized representation of the value
+Returns a String or Binary with the serialized representation of the value
```

Comment:
TODO: copyedit. determine whether we want upper or lower case to refer to a value that is a string or binary. Lower is prob more common. And fix "Json" below to 'JSON' and prob fix "MultiPart"